packages/modules/src/scoring/Scoring.ts

```diff
@@ -0,0 +1,66 @@
+import { SCORE_MODULUS, IJournalContent } from '@cord.network/types'
+import { Crypto } from '@cord.network/utils'
+import {
+  HexString,
+  SCORE_IDENT,
+  SCORE_PREFIX,
+  ScoreType,
+  IScoreAverageDetails,
+} from '@cord.network/types'
+import { Identifier } from '@cord.network/utils'
+import { ConfigService } from '@cord.network/config'
+import type { H256 } from '@polkadot/types/interfaces'
+import { getHashRoot } from '../document/Document'
+import { fetchScore } from './Scoring.chain'
+
+export function adjustAndRoundRating(rating: number): number {
+  rating = Math.round(rating * SCORE_MODULUS)
+  return rating
+}
+
+export function ComputeActualRating(rating: number): number {
+  return rating / SCORE_MODULUS
+}
+
+export function ComputeAverageRating(rating: number, count: number): number {
+  return rating / count
+}
+
+export function generateRootHashFromContent(journalContent: IJournalContent) {
```

Comment:
We don't require selective disclosure here. One hash for the content is good enough.